FreeOTP.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		2489072124910C89001C75A9 /* UIFont+Ext.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2489072024910C89001C75A9 /* UIFont+Ext.swift */; };
```

Comment:
I think it would be nice to move `Color.swift` and `UIFont+Ext.swift` into a file named more generically, like `Style.swift`, containing the code from both these files.

Reply:
Sure :)